backend/apps/progress/tests/test_leaderboard_concurrency.py

```
@@ -0,0 +1,72 @@
+            try:
+                with transaction.atomic():
+                    XPEvent.objects.create(
+                        user=user,
+                        source_type="lesson",
+                        source_id=1,
+                        xp_delta=xp,
+                        base_points=xp,
+                        multiplier_applied=1.0,
+                    )
+            except Exception as e:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)